fileconnector/fileconnector-2.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileSearch.java

```diff
@@ -100,4 +123,75 @@ private void search(String source, String filePattern, MessageContext messageCon
             }
         }
     }
+
+    /**
+     *
```

Comment:
Shall we explain what this class does ?

Reply:
Fixed in https://github.com/wso2-extensions/esb-connector-file/pull/28